packages/react-devtools-timeline/src/TimelineNotSupported.js

```diff
@@ -12,14 +12,46 @@ import {isInternalFacebookBuild} from 'react-devtools-feature-flags';
 
 import styles from './TimelineNotSupported.css';
 
-export default function TimelineNotSupported(): React.Node {
+type Props = {
+  isPerformanceTracksSupported: boolean,
+};
+
+function PerformanceTracksSupported() {
   return (
-    <div className={styles.Column}>
-      <div className={styles.Header}>Timeline profiling not supported.</div>
+    <>
+      <p className={styles.Paragraph}>
+        <span>
+          Please use{' '}
+          <a
+            className={styles.Link}
+            href="https://react.dev/reference/dev-tools/react-performance-tracks"
+            rel="noopener noreferrer"
+            target="_blank">
+            React Performance tracks
+          </a>{' '}
+          instead of the Timeline profiler.
+        </span>
+      </p>
+    </>
+  );
+}
+
+function UnknownUnsupportedReason() {
+  return (
+    <>
       <p className={styles.Paragraph}>
         <span>
           Timeline profiler requires a development or profiling build of{' '}
-          <code className={styles.Code}>react-dom@^18</code>.
+          <code className={styles.Code}>react-dom@{'>='}18</code>. React 19.2
+          and above must use{' '}
+          <a
+            className={styles.Link}
+            href="https://react.dev/reference/dev-tools/react-performance-tracks"
+            rel="noopener noreferrer"
+            target="_blank">
+            React Performance tracks
+          </a>{' '}
+          instead.
```

Comment:
- i'd replace "must" with "should" 
- would wrap the "react 19.2 ..." message to a new line
- maybe say something about deprecating this and removing in a future release